web/src/stream_list.ts

```diff
@@ -747,7 +747,7 @@ export function update_stream_sidebar_for_narrow(filter: Filter): JQuery | undef
     }
 
     if (!info.topic_selected) {
-        $stream_li.addClass("active-filter");
+        $stream_li.addClass("top-left-active-filter");
```

Comment:
I don't know why you're renaming `active-filter` here; the stream list is a "bottom-left" item, so if you're going to rename or split this, it should be a separate `bottom-left-active-filter` class.